src/parser/rogue/outlaw/modules/spells/SliceAndDiceUptime.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'parser/core/Analyzer';
+import Statistic from 'interface/statistics/Statistic';
+import { formatPercentage } from 'common/format';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import SpellLink from 'common/SpellLink';
+
+class SliceAndDiceUptime extends Analyzer {
+  get percentUptime() {
+    return (
+      this.selectedCombatant.getBuffUptime(SPELLS.SLICE_AND_DICE.id) / this.owner.fightDuration
+    );
+  }
+
+  statistic() {
+    return (
+      <Statistic
+        position={STATISTIC_ORDER.CORE(100)}
+        size="flexible"
+      >
+        <div className="pad">
+          <label><SpellLink id={SPELLS.SLICE_AND_DICE.id} /> Uptime</label>
```

Comment:
Dont believe you need this <label>

Reply:
![image](https://user-images.githubusercontent.com/23483002/95269673-93a43800-07ff-11eb-9c92-d683c0154802.png)
Label is needed so user knows what the percentage is for.

Reply:
Ahh, havent seen anyone do it that way.

```
<BoringSpellValueText spell={SPELLS.SLICE_AND_DICE}>
  <>
    <UptimeIcon /> {formatPercentage(this.uptime)}% <small>Uptime</small>
  </>
</BoringSpellValueText>
```

^That is what i usually do, but idk that there is an issue with doing it your way. it might be fine.